packages/contracts/contracts/router/modules/exchanges/Permit2Module.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {BaseExchangeModule} from "./BaseExchangeModule.sol";
+import {BaseModule} from "../BaseModule.sol";
+import {IAllowanceTransfer} from "../../../interfaces/IAllowanceTransfer.sol";
+
+contract Permit2Module is BaseExchangeModule {
+    // --- Fields ---
+
+    IAllowanceTransfer public constant PERMIT2 =
+        IAllowanceTransfer(0x000000000022D473030F116dDEE9F6B43aC78BA3);
+
+    // --- Constructor ---
+
+    constructor(address owner, address router)
+        BaseModule(owner)
+        BaseExchangeModule(router)
```

Comment:
No need to inherit from `BaseExchangeModule`, `BaseModule` should be enough since we don't need the router for anything.